src/material/stepper/stepper.md

```diff
@@ -123,7 +123,7 @@ linear stepper) and the user has interacted with the step. The user, however, ca
 this default `completed` behavior by setting the `completed` attribute as needed.
 
 #### Overriding icons
-By default, the step headers will use the `create` and `done` icons from the Material design icon
+By default, the step headers will use the `edit` and `done` icons from the Material design icon
```

Comment:
I think that the current text is correct. See https://github.com/angular/components/blob/master/src/material/stepper/step-header.ts#L139

Reply:
Please check out the following example where I am overwriting the `edit` as well as the `create` icons with the specific text. For the active, untouched step, the edit icon is used:
https://stackblitz.com/edit/angular-q4sfkg?file=src/app/stepper-states-example.html

```
  <ng-template matStepperIcon="edit">
    <span>EDIT</span>
  </ng-template>
  <ng-template matStepperIcon="create">
    <span>CREATE</span>
  </ng-template>
```

Reply:
The confusion here comes from the fact that the step state is `edit`, but the icon it is using to represent it is called `create` so I think that the docs are correct. In your example, when you set `matStepperIcon="edit"`, it means that the particular icon will be used for the `edit` state, not that it'll use an icon called `edit`.

Reply:
I see; in that case it is misleading; as the documentation clearly states, by default the icons are used as states:

https://github.com/angular/components/blame/master/src/material/stepper/stepper.md#L138